manifests/TexasInstruments/TIConnectCE/5.4.yaml

```
@@ -0,0 +1,29 @@
+Id: TexasInstruments.TIConnectCE
+Publisher: Texas Instruments
+Name: TI Connect CE
+AppMoniker: ticonnectce
+Version: 5.4
+License: Copyright (c) 1995-2020 Texas Instruments Incorporated
+LicenseUrl: https://education.ti.com/en/software/details/en/13312f7cec074a2dafd7ee5646129839/swticonnectcesoftwareforwindows#show=eula
+MinOSVersion: 10.0.0.0
+Description: Easily take screen captures, transfer files, and write TI basic programs for the TI-84 Plus family.
+Homepage: https://education.ti.com/en/software/details/en/13312f7cec074a2dafd7ee5646129839/swticonnectcesoftwareforwindows
+Tags: ti, connect, calculator, math
+
+Installers:
+  - Arch: x64
+    InstallerType: exe
+    Url: https://d13g8g4x8s9rc1.cloudfront.net/downloads/files/cdn/ti-connect/5.4/TIConnectCE-5.4.0.1181.exe
+    Sha256: 36bceb5dab6b1190a600a7350284df79a3c9b9e0b9accfbecdc05e3f5acc9f63
+    Scope: user
+    Switches:
+      Silent: /S
+      SilentWithProgress: /S
+  - Arch: x64
+    InstallerType: msi
+    Url: https://d13g8g4x8s9rc1.cloudfront.net/downloads/files/cdn/ti-connect/5.4/TIConnectCE-5.4.0.1181.msi
+    Sha256: a0f9cd88e2e1b4c914c5e107d7ed9490a7cd287e3e47de4dd93b8d2da060ba7f
+    Scope: user
+    Switches:
+      Silent: /S
+      SilentWithProgress: /S
```

```suggestion
Installers:
  - Arch: x64
    InstallerType: msi
    Url: https://d13g8g4x8s9rc1.cloudfront.net/downloads/files/cdn/ti-connect/5.4/TIConnectCE-5.4.0.1181.msi
    Sha256: a0f9cd88e2e1b4c914c5e107d7ed9490a7cd287e3e47de4dd93b8d2da060ba7f
```

User scope is the default
Only one or the other for right now, msi doesn't need switches so I assume it is the preferred.

---

Thanks